test/forms/mappings/LocalDateFormatterSpec.scala

```diff
@@ -194,6 +194,7 @@ trait SetUp {
   val dayMsgKey = "cf.form.error.start.date.invalid.day"
   val monthMsgKey = "cf.form.error.start.date.invalid.month"
   val yearMsgKey = "cf.form.error.start.date.invalid.year"
+  val validDateMsgKey = "cf.form.error.start.date.invalid.real-date"
```

Comment:
Better to rename it to invalidDateMsgKey  or dateMsgKey as this key will be used for invalid date.